ui/src/app/job-details/tasks/tasks.component.html

```diff
@@ -3,7 +3,14 @@
     <p class="title">Tasks</p>
     <md-tab-group>
       <md-tab label = "List view"></md-tab>
-      <md-tab label = "Graph view"></md-tab>
+      <!--<md-tab label = "Graph view"></md-tab>-->
+      <md-tab>
+        <ng-template md-tab-label>
+          <a style="all: unset" href="{{ serverUrl == '' ? '/jobs/' + parentJobId : serverUrl + parentJobId + '/timing' }}" target="_blank">
```

Comment:
I would move this logic into a function inside of the tasks component and call it in the template to clean this up a bit